db.go

```diff
@@ -2068,13 +2086,23 @@ func (d *DB) EstimateDiskUsage(start, end []byte) (uint64, error) {
 					)
 				}
 				if err != nil {
-					return 0, err
+					return 0, 0, 0, err
+				}
+				meta, err := d.objProvider.Lookup(fileTypeTable, file.FileBacking.DiskFileNum)
+				if err != nil {
+					return 0, 0, 0, err
+				}
+				if meta.IsShared() {
+					remoteSize += file.Size
```

Comment:
This will be `size` not file.Size, as this is the partial overlap case and `size` is being returned by `reader.EstimateDiskUsage`.

Reply:
hm. I yeah, I suppose. I was thinking in terms of our download, and we need to download the whole file, but I guess we'll be passing the whole span, so we're never in this overlap case anyway, so yeah, `size`. 

Done.